manifests/resource/server.pp

```diff
@@ -25,6 +25,7 @@
 #   [*proxy*]                      - Proxy server(s) for the root location to connect to.  Accepts a single value, can be used in
 #     conjunction with nginx::resource::upstream
 #   [*proxy_read_timeout*]         - Override the default the proxy read timeout value of 90 seconds
+#   [*proxy_send_timeout*]         - Override the default the proxy read timeout value of 90 seconds
```

Comment:
I'm assuming this should read: "Override the default proxy send timeout value of 90 seconds"?